docs/apis/plugintypes/blocks/index.mdx

```diff
@@ -0,0 +1,424 @@
+---
+title: Block plugins
+tags:
+    - Blocks
```

Comment:
Does it affect anything? Could be change it for 4 spaces to follow the Moodle coding style?

I mean, having a 2 spaces indentation at the beginning of the document makes everything harder in editors like vscode (because they detect indentation from the code). The rest of the document uses a 4 spaced indentation, specially in the code examples.

Reply:
I fix it but I wonder if we should change this coding style

Reply:
The frontmatter is yaml. All of the rest of the code is 2 spaces, except for Moodle coding examples I think..?